gn3/app.py

```diff
@@ -27,4 +28,5 @@ def create_app(config: Union[Dict, str, None] = None) -> Flask:
             app.config.from_pyfile(config)
     app.register_blueprint(general, url_prefix="/")
     app.register_blueprint(gemma, url_prefix="/gemma")
+    app.register_blueprint(correlation)
```

Comment:
**issue:** You should probably use a url prefix here. So that a user hits: _localhost:8080/correlation/_